### PR TITLE
Update app.js

### PR DESCRIPTION
**Pull request by jabiers**
if i want to add multiple parents like "  Circles.registerCircle('can delete content', ['authenticated','admin']);  "

it will make error

because it makes  " circles : [ [ 'authenticated','admin' ] ] "

### CODE REVIEW

**Comment by ADSKLowenthal** [comment]
@liorkesos @VikramTiwari Circles isn't part of Mean now, we could close this.

**Comment by VikramTiwari** [comment]
Not related now.

File: packages/core/circles/app.js [hunk 1]
```diff
@@ -47,8 +47,7 @@ function registerCircle(name, parents) {
   var query = { name: name };
   var set = {};
   if(parents) {
-    set.$push = {
-      circles: parents
+      set.circles = parents;
```
Comment (timelf123) [inline packages/core/circles/app.js]: note: this fix among others still applies to meanio-circles when the issue queue is opened again

**Comment by timelf123** [comment]
https://github.com/linnovate/mean/pull/1490/commits/952f4887f924f063f6311b4b36e2c56458c2ab92#r78624079 need to make a new issue in meanio-circles as I ran into this as well

**Pull request closed without merging**